contrib/forms/ippf_srh/new.php

```diff
@@ -76,12 +76,12 @@ function end_group()
 
     if ($formid) {
         // Updating an existing form.
-        $query = "UPDATE form_ippf_srh SET $sets WHERE id = '$formid'";
-        sqlStatement($query);
+        $query = "UPDATE form_ippf_srh SET ? WHERE id = ?";
```

Comment:
note that $sets is not simply a variable
(the good thing is that it is already escaped, so you can just keep the $sets there)

Reply:
TODO for me

Reply:
done